roles/xmpp/tasks/prosody.yml

```diff
@@ -37,6 +34,23 @@
   template: src=prosody.cfg.lua.j2 dest=/etc/prosody/prosody.cfg.lua group=root owner=root
   notify: restart prosody
 
+- name: Copy SSL private key and cert
```

Comment:
Not as it stands.  The LE client sets up /etc/letsencrypt with root.root ownership.  We'd have to whack at least a portion of the tree recursively to root.ssl-cert.